benchmarks/src/main/java/org/elasticsearch/benchmark/compute/operator/BlockBenchmark.java

```diff
@@ -141,7 +142,7 @@ static Block[] buildBlocks(String dataType, String blockKind, int totalPositions
                             blocks[blockIndex] = new BooleanBigArrayBlock(
                                 valuesBigArray,
                                 totalPositions,
-                                null,
+                                IntStream.rangeClosed(0, totalPositions).toArray(),
```

Comment:
These were required to run the self-test with assertions enabled. Which we do when we run them in tests.